### ISSUE TITLE
disable calender / hide from menu

### ISSUE THREAD

**Issue by hmr-it-jr** [kevinpapst/kimai2#2009]
**Is your feature request related to a problem? Please describe.**
Is there a way to disable / hide the calender from the menu?

I've looked here https://www.kimai.org/documentation/calendar.html and here .../de/admin/permissions

**Describe the solution you'd like**
If it's currently not possible add a permission to it like it exists for invoices so it can be disabled in the permissions menu.

**Comment by kevinpapst** [comment]
Yes, it's up to the user to choose which way he wants to use for time-tracking.
What about CustomCSSBundle => `ul.sidebar-menu li#calendar { display:none; }`?

**Comment by hmr-it-jr** [comment]
Totally fine with that!

**Comment by kevinpapst** [comment]
👍
